cmd/tnf/claim/show/failures/failures.go

```diff
@@ -291,7 +292,7 @@ func showFailures(_ *cobra.Command, _ []string) error {
 	}
 
 	// Check claim format version
-	err = claim.CheckClaimVersion(claimScheme.Claim.Versions.ClaimFormat)
+	err = common.CheckClaimVersion(claimScheme.Claim.Versions.ClaimFormat)
```

Comment:
Thanks, good catch! Not sure what happened there. I created the common.go because of circular dependency, but it was clearly not needed in this version of the code (in addition to this the function was duplicated). I might have done this when trying to reuse the claim schema repo un-marshalling functions. I will keep the function in the claim module.